src/components/TodoListItem/index.jsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import {
+  ListGroup, InputGroup, FormControl, Button, Form, Jumbotron, Container,
+} from 'react-bootstrap';
+import {
+  changeStatus, deleteTodo, addDescription,
+} from '../../actions';
+import availableStatus from '../../ultil';
+
+// eslint-disable-next-line max-len
+const selectTodoById = (state, todoId) => Object.values(state.todos.tasks).find((todo) => todo.id === todoId);
+
+function TodoListItem({ id }) {
+  const todo = useSelector((state) => selectTodoById(state, id));
+
+  const { text, status, description } = todo;
+  const [descriptionText, setDescriptionText] = useState('');
+  const dispatch = useDispatch();
+
+  // Add description
+  const handleDescriptionChange = (e) => {
+    setDescriptionText(e.target.value);
+  };
+
+  const handleEnterAddDes = (e) => {
+    const trimedText = descriptionText.trim();
+    if (e.which === 13 && trimedText) {
```

Comment:
I dont know what is this and why have this logic, Could you describe for me?

Reply:
That's is made for action when i press "Enter" key ( on Input "What needs to be done?" ). When i design again, i removed it. i will commit and push it as soon as possible.